### ISSUE TITLE
unable to install xmpp because mongodb is missing

### ISSUE THREAD

**Issue by si458** [tiredofit/docker-freepbx#39]
when you try installing the xmpp extension it will fail because mongodb is missing

is it possible to include this in the image at all?
```
root@dc50f5977496:/home/asterisk# fwconsole ma downloadinstall xmpp
No repos specified, using: [standard] from last GUI settings
Downloading module 'xmpp'
Processing xmpp
Verifying local module download...Verified
Extracting...Done
Download completed in 2 seconds
MongoDB is not installed
Unable to install module xmpp:
 - Failed to run installation scripts
Updating Hooks...Done
```

**Comment by tiredofit** [comment]
There was one release (over a year ago) that I did include it, but then stopped for reasons I can't quite remember. We swtiched to Rocketchat instead. I can (re)add it in over the next few days. It _should_ be as simple as exporting `/var/lib/mongodb` however I'd like to get it in the folder structure I have defined (/data) for a cleaner install. Let me see what I can put together.

**Comment by si458** [comment]
im just looking if i can do it myself as all i did was `apt-get install mongodb` in the container,
im now also trying to see if its working correctly as i cant see mongodb running in the process list?

**Comment by tiredofit** [comment]
Try installing XMPP again, we might need to build it into the startup script, I'm not entirely sure if the "fwconsole restart" command will fire up the process.

**Comment by si458** [comment]
ok so been able to get it working :) 

1. apt-get install mongodb (best during the init stage at start create)
2. systemctl mongodb start (needs to be started when container starts, at sametime or just after mysql is started)
3. map the /var/lib/mongodb folder(ln -s /data/var/lib/mongodb /var/lib/mongodb)
4. fwconsole ma downloadinstall xmpp (OPTIONAL INSTALL XMPP)
5. fwconsole restart (needed to start the letschat after OPTIONAL INSTALL)

**Comment by tiredofit** [comment]
To be included in upcoming `3.2` release

**Comment by tiredofit** [comment]
Released!
